config/benchmarks/scenarios/test/reads_1s.yaml

```diff
@@ -0,0 +1,17 @@
+scenarios:
```

Comment:
Granted, the tests are currently turned off because of the desync in the `cluster-logging-load-client` builds, but the difference is just that these scenarios would just be used for testing purposes and the ranges provided by both are no significant enough to put into the testing cycle.